README.Rmd

```diff
@@ -15,11 +15,12 @@ knitr::opts_chunk$set(
 
 # goodpractice <img src="man/figures/logo.png" align="right" width="20%" height="20%" />
 
-[![Linux Build Status](https://travis-ci.org/MangoTheCat/goodpractice.svg?branch=master)](https://travis-ci.org/MangoTheCat/goodpractice)
-[![Windows Build status](https://ci.appveyor.com/api/projects/status/github/MangoTheCat/goodpractice?svg=true)](https://ci.appveyor.com/project/gaborcsardi/goodpractice)
-[![](https://www.r-pkg.org/badges/version/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
-[![CRAN RStudio mirror downloads](https://cranlogs.r-pkg.org/badges/goodpractice)](https://www.r-pkg.org/pkg/goodpractice)
```

Comment:
Any particular reason to remove this badge?

Reply:
Oh!! No, that's removed in error - it did catch my eye yesterday but I promptly forgot about it 😁 I'll reinstate it now!